pkg/handler/processor/processor.go

```diff
@@ -76,6 +76,7 @@ const (
 	FormatJSON      FormatType = "JSON"
 	FormatJSONLines FormatType = "JSON_LINES"
 	FormatXML       FormatType = "XML"
+	FormatBzip2     FormatType = "json.bz2"
```

Comment:
I wouldn't mix file suffix with the existing constants (and it's related to encoding, not format). Maybe introduce a map that will do this transformations

bz2-> Encoding.BZIP
zst -> Encoding.ZSTD

Then a single lookup could do the trick.

Reply:
Sounds good, refactored with this change

Reply:
Thanks. LGTM now